cmd/status/cmdstatus.go

```diff
@@ -374,3 +372,42 @@ func (ir *InventoryLoader) GetInvInfo(cmd *cobra.Command, args []string) (invent
 	inv := inventory.WrapInventoryInfoObj(invObj)
 	return inv, nil
 }
+
+func (r *Runner) listClusterInventoryObjs(ctx context.Context) (map[string]object.ObjMetadataSet, error) {
+	dc, err := r.factory.DynamicClient()
+	if err != nil {
+		return nil, err
+	}
+	mapper, err := r.factory.ToRESTMapper()
+	if err != nil {
+		return nil, err
+	}
+	// Define the mapping
+	mapping, err := mapper.RESTMapping(inventory.ConfigMapGVK.GroupKind(), inventory.ConfigMapGVK.Version)
+	if err != nil {
+		return nil, err
+	}
+
+	// retrieve the list from the cluster
+	clusterInvs, err := dc.Resource(mapping.Resource).List(ctx, metav1.ListOptions{})
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil, err
+	}
+	if apierrors.IsNotFound(err) {
```

Comment:
nit: This pattern works, and technically has less cyclomatic complexity, but i usually find it more readable to do the IsNotFound check inside the err != nil clause.

Reply:
ah, i see you just moved and renamed this. NBD.